### PR TITLE
Bug 12345 - Fix that annoying bug R=TatianaMigacheva

### PR DESCRIPTION
**Pull request by TatianaMigacheva**
Our first commit for feature 12345
@TatianaMigacheva
@harman-red-square/reviewers-test

### CODE REVIEW

File: Android.mk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 LOCAL_PATH:= $(call my-dir)
-
+#test changes
```
Comment (TatianaMigacheva) [inline Android.mk]: please check spelling

**Comment by TatianaMigacheva** [comment]
please start review the new feature

File: Android.mk [hunk 2]
```diff
@@ -1,5 +1,5 @@
 LOCAL_PATH:= $(call my-dir)
```
Comment (TatianaMigacheva) [inline Android.mk:1]: check #2 before implementing this fix 
see for details at [test.txt](https://github.com/harman-red-square/gaia/files/7512/test.txt)

**Comment by harman-red-square** [comment]
why notification emais are not sent???

**Comment by TatianaMigacheva** [comment]
testing email notification of reviewers

**Comment by TatianaMigacheva** [comment]
testing email notification of reviewers @harman-red-square/reviewerstest

**Comment by TatianaMigacheva** [comment]
adding reviewers to first comment, check email notification

File: Android.mk (hunk 1 above)
Comment (TatianaMigacheva) [inline Android.mk]: test email notification in line note

**Comment by TatianaMigacheva** [comment]
test new comment notification

**Comment by TatianaMigacheva** [comment]
@tkalpina

File: Android.mk (hunk 1 above)
Comment (TatianaMigacheva) [inline Android.mk]: test email notification for bug #1

**Pull request merged**